editor-extensions/vscode/package.json

```diff
@@ -2,15 +2,15 @@
   "name": "vscode-grain",
   "displayName": "Grain",
   "description": "Grain support for Visual Studio Code.",
-  "version": "0.15.0",
+  "version": "0.16.0",
```

Comment:
release-please handles bumping this, correct?

Reply:
I'm not sure, paging @ospencer

Reply:
```suggestion
  "version": "0.15.0",
```